src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props

```diff
@@ -472,7 +472,7 @@
 		</ItemGroup>
 	</Target>
 
-	<Target Name="_WPFIssue6792_RemoveDuplicateAnalyzers" BeforeTargets="CoreCompile">
+	<Target Name="_WPFIssue6792_RemoveDuplicateAnalyzers" BeforeTargets="CoreCompile" Condition="'$(UnoUseWPFIssue6792Workaround)'=='true'">
```

Comment:
Why not remove the target altogether?

Reply:
Because people still running with intermediate versions may still have the issue and editing targets files is not particularly nice. We'll remove it later on.